docs/html/1-intro/README.md

```diff
@@ -0,0 +1,116 @@
+# What is Web Development & HTML
+
+Today we cover what web development is and HTML basics. In this section of the worksheet, you will create your own _index.html_ page from the ground up. The _index.html_ file is the main HTML for a web page. HTML is one of the building blocks of a web page. It defines the structure of the page and lays the foundation to then add styles and dynamic content.
+
+At the end of this section, you will have a webpage that looks like this:
+
+![](./images/endResult.png ":class=image-border")
+
+> [!WARNING]
+> Before starting the worksheet, please take a moment to review the [Setup instructions](../setup/?id=setup) to ensure you have all the tools and workspace setup you need for today's work.
+
+> [!WARNING]
+> If you are using StackBlitz, please skip to [StackBlitz instructions](./?id=cloud-ide) below.
+
+# Create the HTML file
+
+<!-- [filename](./1file.md ':include') -->
+<!-- Contents from file. make changes in external files and copy here until bug is fixed -->
```

Comment:
This bug was fixed by docsify, so you can load markdown from external files now. Please feel to utilize this feature as you'd like. :)

Reply:
gotcha. I left those comments from the previous worksheet. I don't think I need those so probably delete them.